webapp-django/crashstats/crashstats/templates/crashstats/home.html

```diff
@@ -59,18 +99,16 @@ <h2 id="homepage-heading">{{ product }} {% if version %}{{ version }}{% endif %}
 
 {% endblock %}
 {% block site_js %}
-  {{ super() }}
-  <!--[if IE]><script type="text/javascript" src="{{ static('crashstats/js/flot-0.7/excanvas.pack.js') }}"></script><![endif]-->
-  {% compress js %}
-  <script src="{{ static('crashstats/js/lib/d3.min.js') }}"></script>
-  <script src="{{ static('crashstats/js/lib/metricsgraphics.min.js') }}"></script>
-  {% endcompress %}
+    {{ super() }}
+    <!--[if IE]><script type="text/javascript" src="{{ static('crashstats/js/flot-0.7/excanvas.pack.js') }}"></script><![endif]-->
```

Comment:
Bad news: it's still being used in 3 places, explosive crashes, status and topcrasher.